server/src/main/java/org/elasticsearch/action/support/IndicesOptions.java

```diff
@@ -1321,6 +1392,14 @@ public static IndicesOptions strictExpandOpen() {
         return STRICT_EXPAND_OPEN;
     }
 
+    /**
+     * @return indices options that requires every specified index to exist, expands wildcards only to open indices and
+     * allows that no indices are resolved from wildcard expressions (not returning an error).
+     */
+    public static IndicesOptions strictExpandOpenIncludeFailureStore() {
+        return STRICT_EXPAND_OPEN_FAILURE_STORE;
+    }
```

Comment:
Well, I did not like it initially, but I do understand they make things a bit more readable and even more traceable, since you can look for all the places where a certain configuration is used.